cdap-integration-test/src/main/java/co/cask/cdap/test/remote/RemoteApplicationManager.java

```diff
@@ -192,15 +181,15 @@ public boolean isRunning(ProgramId programId) {
   @Override
   public List<RunRecord> getHistory(Id.Program programId, ProgramRunStatus status) {
```

Comment:
there isn't, but there should be. These methods that delegate should actually be in the abstract class too